homeassistant/helpers/integration_platform.py

```diff
@@ -119,14 +165,16 @@ async def async_process_integration_platforms(
         hass.bus.async_listen(
             EVENT_COMPONENT_LOADED,
             partial(
-                _async_process_integration_platforms_for_component,
+                _async_integration_platform_component_loaded,
                 hass,
                 integration_platforms,
             ),
+            run_immediately=True,
         )
     else:
         integration_platforms = hass.data[DATA_INTEGRATION_PLATFORMS]
 
+    async_register_preload_platform(hass, platform_name)
```

Comment:
This is called twice

Reply:
Bad merge.  Will fix it all up when I merge in dev after the break off merges

Reply:
b0d9b2a54b4